template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApp-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Microsoft",
+    "classifications": [ "Common", "Console" ],
+    "name": "Console Application",
+    "generatorVersions": "[1.0.0.0-*)",
+    "description": "A project for creating a command-line application that can run on .NET Core on Windows, Linux and macOS",
+    "groupIdentity": "Microsoft.Common.Console.App",
+    "precedence": "1000",
+    "identity": "Microsoft.Common.Console.App.CSharp.6.0",
+    "shortName": "console-app",
+    "tags": {
+        "language": "C#",
+        "type": "project"
+    },
+    "sourceName": "Company.ConsoleApplication1",
+    "preferNameDirectory": true,
+    "symbols": {
+        "target-framework-override": {
```

Comment:
I renamed the symbols to avoid extra lines in `dotnetcli.host.json`, let me know if time is good.